src/pages/Dashboard.js

```diff
@@ -31,7 +31,13 @@ const yourFamilyRows = ({ maritalStatus, children }) => {
 }
 
 const yourIncomeRows = ({ income }) => {
```

Comment:
I agree that I like this style better. Is the `= {}` a default value that's passed to the function?

Reply:
Yeah, if there's no value passed in (ie, you call `yourIncomeRows()`), then you default `income` to an empty object.

Otherwise, `income` will default to `undefined`, and trying to access `income.employerName` will cause runtime errors.